Pipfile

```diff
@@ -0,0 +1,8 @@
+[[source]]
+url = "https://pypi.python.org/simple"
+verify_ssl = true
+
+[packages]
+python-snappy = "*"
```

Comment:
Can you add minimum version requirements? 

* python-snappy >= 0.5
* kafka-python >= 1.3.0
* PyYAML >= 3.11